src/main/java/duke/duke/Parser.java

```diff
@@ -0,0 +1,28 @@
+package duke.duke;
+public class Parser {
+
+    public Parser() {
```

Comment:
I think this empty constructor is redundant! Maybe you can consider removing it to reduce the LoC of your program?